src/io/matpower.jl

```diff
@@ -504,8 +506,8 @@ function mp2pm_dcline(data::Dict{String,Any})
         loss0 = dcline["loss0"]
         loss1 = dcline["loss1"]
 
-        delete!(dcline, "pmin")
-        delete!(dcline, "pmax")
+#        delete!(dcline, "pmin")
+#        delete!(dcline, "pmax")
```

Comment:
I think leaving these in will risk confusion on which parameter PowerModels is actually using.  I recommend renaming to `mp_pmin` and `mp_pmax` to mitigate confusion.